ballista/rust/client/README.md

```diff
@@ -17,6 +17,102 @@
   under the License.
 -->
 
-# Ballista - Rust
+# Ballista: Distributed Scheduler for Apache Arrow DataFusion
```

Comment:
have you considered using `#![doc = include_str!("../README.md")]` here to keep both versions in sync? we could annotate the example code with `rust,no_run` to make sure it's skipped in doc test and also renders properly on github.

Reply:
I had no idea about this feature ... thank you for pointing this out. I will update the PR tomorrow with this.